css/silverstripe-navigator.css

```diff
@@ -4,6 +4,10 @@ div#SilverStripeNavigator {
     border-top: 0;
 }
 
-#SilverStripeNavigatorMessage {
+div#SilverStripeNavigatorMessage {
```

Comment:
Needed to increase specificity to override the `top` property. This is consistent with the other selector above.